proxy/http/HttpBodyFactory.h

```diff
@@ -100,6 +100,22 @@ class HttpBodyTemplate
   char *template_pathname;
 };
 
+////////////////////////////////////////////////////////////////////////
+//
+//      class HttpBodySetRawData
+//
+//      Raw data members of HttpBodySet
+//
+////////////////////////////////////////////////////////////////////////
+
+struct HttpBodySetRawData {
```

Comment:
As long as this is being updated, add in line initializers. If there's a "bad" magic value, initialize `magic` to that.

Reply:
Yeah, I'll add `unsigned int magic = 0` to it.